test/fixtures/attr-reflection-app.js

```diff
@@ -1,10 +1,11 @@
-import { Component, h } from '../../lib';
+import {Component, h} from '../../lib';
 
 export class AttrReflectionApp extends Component {
   get config() {
     return {
-      template: state => h('div', {class: {'attr-app': true}}, [
-        h('p', `Value of attribute wombats: ${this.getAttribute('wombats')}`),
+      // eslint-disable-next-line no-unused-vars
```

Comment:
It's just a test fixture, might as well make it conform with `template: () => h(...)` (ditto for the other fixtures below)